app/src/routes/routesMap.js

```diff
@@ -350,8 +359,7 @@ export const onBeforeRouteChange = (dispatch, getState, { action }) => {
     (isChangedProject || isAdminCurrentPageType) &&
     !isAdminNewPageType
   ) {
-    if (isProjectExists) {
-      dispatch(fetchOrganizationBySlugAction(hashOrganizationSlug));
+    if (hasPermission || isProjectExists) {
```

Comment:
Will this logic handle the admin access to a not assigned organization? Or it will be processed separately?

Reply:
This logic handle the admin access to a not assigned organization.